sirepo/package_data/static/js/sirepo.js

```diff
@@ -1028,8 +1028,12 @@ SIREPO.app.factory('timeService', function() {
     const UNIX_TIMESTAMP_SCALE = 1000;
     const self = {};
 
+    self.getCurrentUnixTime = () => {
+	return self.getUnixTime(new Date());
+    };
+
     self.getUnixTime = (date) => {
```

Comment:
This is converting to unixTime so I would call it "toUnixTime" or just "unixTime".

Reply:
2fbd304ef